index.html

```diff
@@ -6,6 +6,175 @@
     <title>Homework</title>
 </head>
 <body>
-    
+    <header>
+        <section>
+            <nav>
+                <div>
+                    <img src="icons/Logo.svg" alt="logo" srcset="">
+                </div>
+            </nav>
+        </section>
+        <section>
+            <ul>
+                <li>Product</li>
+                <li>Customers</li>
+                <li>Pricing</li>
+                <li>Resources</li>
+            </ul>
+        </section>
+        <section>
+            <a href="#">Sign In</a>
+            <a href="#">Sign Up</a>
+        </section>
```

Comment:
а ось тут потрібен `nav`, оскільки він показує,. що в цьому місці навігація
```suggestion
        <nav>
            <a href="#">Sign In</a>
            <a href="#">Sign Up</a>
        </nav>
```

Reply:
тег `section` зазвичай має містити блок сайту з контентом, в якому бажано має бути заголовок (бажано але не обов'язково)

https://html.spec.whatwg.org/multipage/sections.html#the-section-element